### DISCUSSION TITLE
Auto Delete cookies for certan websites  ?

### DISCUSSION THREAD

**Discussion by ghost** [qutebrowser/qutebrowser#7223] [Q&A] [answered]
for example I don't my browser to store any cookies form youtube.com, but store cookies form github.com 
is there any way to achieve this ?

**Comment by mschilli87** [comment, marked as answer]
See https://github.com/qutebrowser/qutebrowser/issues/3636: You can already use URL patterns to decide which websites to *accept* cookies from (`content.cookies.accept`). Which ones to *store* (`content.cookies.store`) is still on the wishlist though.
